main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	logFile := flag.String("log-file", "/var/log/k8s-athenz-istio-auth/k8s-athenz-istio-auth.log", "log file location")
 	logLevel := flag.String("log-level", "info", "logging level")
 	enableAuthzPolicyController := flag.Bool("enable-authzpolicy-controller", true, "enable authzpolicy controller to create authzpolicy dry run resource")
```

Comment:
Should we make this arg `enable-ap-controller` to be consistend with `ap-enabled-list`?